.travis.yml

```diff
@@ -1,17 +1,17 @@
 # Travis configuration for PalMA
 
-sudo: false
-
 language: php
 
+sudo: required
+dist: trusty
```

Comment:
Isn't `sudo: false` (= container based infrastructure) the recommended standard for new projects?